### PR TITLE
move backend to its own folder

### PR DESCRIPTION
**Pull request by harunurhan**

### CODE REVIEW

File: backend/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ ARG POETRY_EXTRA_ARGS=''
 RUN poetry export --without-hashes --format=requirements.txt ${POETRY_EXTRA_ARGS} && \
     pip install -r requirements.txt
 
-COPY setup.py README.md ./
+COPY setup.py ./
```
Comment (drjova) [inline backend/Dockerfile]: I needs ``` README.md `` https://github.com/inspirehep/inspirehep/blob/master/setup.py#L14

Comment (harunurhan) [reply backend/Dockerfile]: why?

Comment (drjova) [reply backend/Dockerfile]: https://github.com/inspirehep/inspirehep/blob/master/setup.py#L14

Comment (drjova) [reply backend/Dockerfile]: if we remove it from Dockerfile we should remove it from the setup.py

Comment (harunurhan) [reply backend/Dockerfile]: your links are for the current master, it's already in the diff.

Comment (drjova) [reply backend/Dockerfile]: my bad, haven't seen the changes in the ``setup.py``

**Pull request merged**